packages/network-controller/tests/NetworkController.test.ts

```diff
@@ -807,7 +807,7 @@ describe('NetworkController', () => {
                       },
                     },
                   ],
-                  stubGetEIP1559CompatibilityWhileSetting: true,
```

Comment:
Previously this option may have been needed if the developer (for some reason) wanted to stub `getEIP1559Compatibility` but not `lookupNetwork`. Now that isn't necessary anymore, because both of the methods that `lookupNetwork` calls internally are private, so the only thing that can be stubbed is `lookupNetwork`.